src/js/tech/tech.js

```diff
@@ -551,7 +558,10 @@ class Tech extends Component {
       // we don't overwrite the injected window.WebVTT if it loads right away
       window.WebVTT = true;
       this.el().parentNode.appendChild(script);
+    } else {
+      this.ready(this.addWebVttScript_);
```

Comment:
Nice, glad to see this change from an event listener. I was going to mention that could cause problems with multiple ready events. 💯